src/aspire/source/image.py

```diff
@@ -704,7 +884,7 @@ def save(
 
     def _populate_common_metadata(
```

Comment:
This only needs to access `n` from `self`, so we may want to consider making this a static/class/external method (it doesn't really belong to the class).

Reply:
Converted to static method.   This and related methods could potentially be collapsed now that we have the metadata a dict already. I'll make an issue.

Reply:
https://github.com/ComputationalCryoEM/ASPIRE-Python/issues/921

Reply:
Thanks.